recipe/meta.yaml

```
@@ -0,0 +1,114 @@
+{% set name = "fairseq2" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/facebookresearch/fairseq2
+  git_rev: v0.2.0
```

```suggestion
  url: https://github.com/facebookresearch/fairseq2/archive/refs/tags/v0.2.0.tar.gz
```
This will allow you drop the dependency on git and get rid of the submodules. Additionally, we will be able to check the hash of the artifact.

---

I still do need some submodules, notably `sentencepiece` which was ultimately the cause of that **segfault**. I can obviously still use the tarball from GitHub but I cannot get rid of the `git` dependency.

---

Unfortunately it seems that the `.gitmodules` file is not included in the tarball so some of the vendored submodules that are necessary cannot be cloned.